Mana-Cores/eoc.json

```diff
@@ -20,77 +20,77 @@
 		"type": "effect_on_condition",
 		"id": "BEGIN_BREAKTHROUGH",
 		"//": "Ensures no bionics, that impurities are low enough, and has a 1 out of 3 chance it succeed",
-		"condition": { "and": [ { "not": {"u_has_bionics": "ANY" } }, { "compare_int": [ { "u_val": "vitamin", "name": "impurities" }, "<=", { "arithmetic": [ { "u_val": "var", "var_name": "manacoretier" }, "*", { "const": 50 } ] } ] }, { "x_in_y_chance": { "x": 1, "y": 3 } } ] },
+		"condition": { "and": [ { "not": {"u_has_bionics": "ANY" } }, { "math": [ "(u_manacoretier*50)<=u_val('vitamin', 'name: impurities')" ]   }, { "x_in_y_chance": { "x": 1, "y": 3 } } ] },

```

Comment:
This looks incorrect, it seems to be saying if the u_manacoretier * 50 <= impurities, when it should be if impurities <= u_manacoretier * 50. In essence the maximum amount of impurities that prevents a breakthrough is 50 * mana core tier. I think it'll work as intended if you swap it.